# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: hs@ox.cx-20130307142016-vcd5rn617jrspyzp
# target_branch: http://svn.twistedmatrix.com/bzr/Twisted/branches\
#   /sslendpoint-certificateoptions-6286/
# testament_sha1: 2b90d8e89286834f86d1a9d6c021c922c0fe9fe6
# timestamp: 2013-03-07 15:20:26 +0100
# base_revision_id: svn-v4:bbbe8e31-12d6-0310-92fd-\
#   ac37d47ddeeb:branches/sslendpoint-certificateoptions-6286:37446
# 
# Begin patch
=== modified file 'twisted/internet/endpoints.py'
--- twisted/internet/endpoints.py	2013-03-06 21:43:58 +0000
+++ twisted/internet/endpoints.py	2013-03-07 14:20:16 +0000
@@ -771,7 +771,7 @@
     keyPEM = FilePath(privateKey).getContent()
     cf = ssl.CertificateOptions(
         privateKey=ssl.PrivateCertificate.loadPEM(
-            keyPEM + certPEM).privateKey.original,
+            certPEM + keyPEM).privateKey.original,
         certificate=ssl.Certificate.loadPEM(certPEM).original, **kw)
     return ((int(port), factory, cf),
             {'interface': interface, 'backlog': int(backlog)})

# Begin bundle
IyBCYXphYXIgcmV2aXNpb24gYnVuZGxlIHY0CiMKQlpoOTFBWSZTWVB0OW4AAZNfgABQUm//91NK
SAC//9/wUANmVoYAgBJJBqYmNQxGpjQmBCDaNIZpBqU9qT1GJ6aQ0YnqZGmI0yDRoyaaBJJoE00I
yptGjKZDQAAaA0MAAAABoAAAAAMAAAABoAAAAAdC+dvb3O+TKF18KQPi7PVG+N2iCdPwzedTMx8d
V/Y30uZnNSdjv1db1wqotcjHKR0aG5A5giJlOWcLHfzIllS1F+0ZJWSxR0N+dzhgKbVSjywW/Ksu
Eg2kLPmyUOdx48wTxF3pU6DwIi39uUJhioMUe4qJdbJeWg0YQdFRsamCThfibsvQVpfO9Ka4XnOe
NxRmy3EBcBYY6XxZHvs9w9yUjAdLrwMEtvCQtRSSZJzRZMomxjQT4EIhMZKwpJPqbQ6DuFmM2RwZ
QYgGQDU4OyoVMEsCH9BrmYMUuV0uYjmPOcBacHHSuk7t3ChyRM1xnlneYPzus5BLUWCuEh9E50br
WVjfxKJOlNK5pAst15LRGBLQvXK21DLek+4xLapZ8MQsMbamE8HyCzRHDPYMoy1aE6OPBNRomOYH
xBidMr+ljcbSyls8V+HDKP8Up5mYZvVuxmRsYaKMq4wdHBgn5xjBk13fexLFr++kVHMKeTVRIxvR
33tzch3Hnnym7XFBLk4OykS4Ml0HkiNoUwS5G0xsT23IlzCvPuL31Ie9g+ZCKvd+UqgoSyYVmLMl
ZYLMlWsSaVPeluSwhgPAf7YHhq74QKkoSHpowmS4eN9+9nOrrC/jIk6Eq8tPglJ+srM/FoL3SM3Z
xUj/F2s5JlBFQ1u1yqpZGYMM5kU4PX16yazLd4aTVscSpRqkeyPnVxgxtN6JbC4I3I6mkaKedWmO
6pDVdO6aXUMiSVOMnC8KOSNq+nKuElgTo3B+1hXRLv8hX52zjNt0h2SCjFQY8/JOiW/40zUdqG7o
jEi+wNMTD5wmhwyAf6TUeV1JDomMmTAyZdk/SB5Ig3qgiJNHEMvhgsIq3zSsjWurMoDy7gK6Y0ki
C4Cy2ZRDvQzon05E+M8WX4FFb5S8RRSnJMuSIJw5JQze5xa3UDTW1EdQq94VhVQQajq4oTh4FWjB
Mi0VqeWO6orGqMAqDplFld+405TVNQowqrcEQvop8xZpNNZ2YaDN4uKtCPaL/i7kinChIKDoctw=
